graphics/sprites/monsters/terranite-mask.xml

```diff
@@ -12,7 +12,7 @@
       <frame index="1" offsetX="-1" offsetY="-40"/>
     </animation>
     <animation direction="right">
-      <frame index="3" offsetX="-1" offsetY="-40"/>
+      <frame index="3" offsetX="1" offsetY="-40"/>
```

Comment:
This... doesn't belongs to this PR O.o

Reply:
yeah that is the problem this gets included in every pull request i make.